functions/config.json

```diff
@@ -0,0 +1,9 @@
+{  
+  "firebase": {
+    "apiKey": "AIzaSyBekh5ZSi1vEhaE2qetH4RU91gHmUmpqgg",
```

Comment:
I think exposing the `apiKey` is safe: http://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public